tool/main.re

```diff
@@ -82,10 +82,10 @@ fn create_base_menu() -> Ui {
             label: Text { text: "New Game Actions" },
             onclick: fn(label: Text) { enter_ui(create_new_game_actions_menu()); },
         }),
-//        UiElement::Button(UiButton {
-//            label: Text { text: "Multiplayer" },
-//            onclick: fn(label: Text) { enter_ui(create_multiplayer_menu()); },
-//        }),
+        UiElement::Button(UiButton {
+            label: Text { text: "Multiplayer" },
+            onclick: fn(label: Text) { enter_ui(create_multiplayer_menu()); },
+        }),
```

Comment:
If there is no server actually running to connect to, I think this should stay commented out in the repo.